test/e2e/pass-opts.feature

```diff
@@ -19,5 +19,4 @@ Feature: Passing Options
     Then it passes with no debug:
       """
       .
-      PhantomJS
```

Comment:
It is not immediately clear to me why my changes cause the `PhantomJS` line not to be printed for this one particular test.